framework/test/unit/modules/interoperability/mainchain/module.spec.ts

```diff
@@ -302,375 +356,532 @@ describe('initGenesisState', () => {
 			});
 		});
 
-		describe('terminatedStateAccounts', () => {
-			it('should not throw error if length of terminatedStateAccounts is zero', async () => {
-				const context = createInitGenesisStateContext(
-					{
-						...genesisInteroperability,
-						// this is needed to verify `validatorsHash` related tests (above)
-						chainInfos: [
-							{
-								...chainInfo,
-								chainData: {
-									...chainData,
-									lastCertificate: {
-										...lastCertificate,
-										validatorsHash: computeValidatorsHash(activeValidators, certificateThreshold),
-									},
-								},
-								chainValidators: {
-									activeValidators,
-									certificateThreshold,
-								},
+		it('should check that _verifyChannelData is called from _verifyChainInfos', async () => {
+			jest.spyOn(interopMod, '_verifyChannelData' as any);
+
+			const context = createInitGenesisStateContext(
+				{
+					...genesisInteroperability,
+					chainInfos: [
+						{
+							...chainInfo,
+							chainData: {
+								...genesisInteroperability.chainInfos[0].chainData,
 							},
-						],
-						terminatedStateAccounts: [],
-					},
-					params,
-				);
+						},
+					],
+				},
+				{
+					...params,
+					header: {
+						timestamp: Date.now(),
+					} as any,
+				},
+			);
 
-				await expect(interopMod.initGenesisState(context)).resolves.not.toThrow();
-			});
+			// doesn't matter what it throw it this stage
```

Comment:
Not sure what this comment means, maybe elaborate or remove it if its not important, also the statement is not correct
`// doesn't matter what it throw at this stage`